beacon_node/store/src/hot_cold_store.rs

```diff
@@ -2210,11 +2210,13 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         ops.push(StoreOp::KeyValueOp(update_blob_info));
 
         self.do_atomically_with_block_and_blobs_cache(ops)?;
-        info!(
-            self.log,
-            "Blob pruning complete";
-            "blob_lists_pruned" => blob_lists_pruned,
-        );
+        if blob_lists_pruned != 0 {
+            info!(
```

Comment:
Alternatively maybe we make it debug (and log it always). This would be consistent with our other database pruning. Sometimes it's nice to be able to see the start and end of a task, e.g. by grepping for `Pruning blobs` and `Blob pruning complete`.

Reply:
That works too!